mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -126,6 +137,35 @@ class LogisticRegressionModel (
       bestClass.toDouble
     }
   }
+
+  override def save(sc: SparkContext, path: String): Unit = {
```

Comment:
We merged multinomial logistic regression. LRModel holds `numFeatures` and `numClasses` now. We need a specialized implementation and a test for it.